android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -203,6 +204,13 @@ class RNPlayerView(
         }
     }
 
+    /**
+     * Make sure this is called after [addPlayerView], otherwise the subtitleView will be overdrawn by the player!
+     */
+    fun addSubtitleView(subtitleView: SubtitleView) {
+        addView(subtitleView)
+    }
```

Comment:
I think it might be worth tracking the view a bit closer and cleanup old view... transforming this from an `add` to a `set`.. similar thing would be appropriate with the `addPlayerView` as well. We hold the view instance explicitly, but still add it blindly to the parent view

Reply:
add cleanup (`removeView`) logic to `playerView` and `subtitleView`:
https://github.com/bitmovin/bitmovin-player-react-native/pull/316/commits/054bc2786ca5588a10bee9d7981bfbbc43336d5a

do you know if there is some additional method call necessary besides `removeView`? :thinking: